search-app/docs/api.md

```diff
@@ -174,6 +174,16 @@ Join the [CMR Client Developer Forum](https://wiki.earthdata.nasa.gov/display/CM
   * [Community Usage Metrics](#community-usage-metrics)
     * [Updating Community Usage Metrics](#updating-community-usage-metrics)
     * [Retrieving Community Usage Metrics](#retrieving-community-usage-metrics)
+  * [Valid KMS Keyword Values](#valid-kms-keyword-values)
+    * [Science Keywords](#kms-science-keywords)
+    * [Platforms](#kms-platforms)
+    * [Instruments](#kms-instruments)
+    * [Projects](#kms-projects)
+    * [Providers](#kms-providers)
+    * [Spatial Keywords](#kms-spatial-keywords)
+    * [Concepts](#kms-concepts)
+    * [Iso Topic Categories](#kms-iso-topic-categories)
+    * [Granule Data Format](#kms-granule-data-format)
```

Comment:
Not sure if there is a different ticket for this but maybe add a reference to the work Leo did with measurement_names.

Reply:
Done